src/componentsEventPage/EventList.js

```diff
@@ -16,9 +16,9 @@ export default function EventList({eventArray, categoryFilter}) {
         <>
             {availableYears.map(year => (
                 <EventContainer key={year}>
-                    <EventYearHeadline>Event {year}</EventYearHeadline>
+                    <EventYearHeadline>Events {year}</EventYearHeadline>
```

Comment:
You have the headline Tag between the ul and li tag

`<ul>
	<h1>
		<li>`

ul elements should contain li elements. Consider to put the headline outside of the ul element.

Reply:
I will try, if this works :)

Reply:
When I put the headline before the ul, its not working because of the JSX, then I have to use more of the empty tags, to wrap the h3 and the ul....